package cms_8120;
option java_package = "com.hikvision.finance.fms.cms.hpp.server";
option java_outer_classname = "RspServerDataProto";
import "type/cmd.proto";

message RspServerData
{
    optional int32 cmd                		= 1;    		// cmd
    optional int32 result                   = 2;    		// 是否成功
    optional int32 data_id					= 3;			//插入数据后返回该记录的id
    optional string error_msg				= 4;			//错误信息               
    
}
